### ISSUE TITLE
accessibility problems

### ISSUE THREAD

**Issue by sukiletxe** [curiousdannii/parchment#19]
Hello,
I'm a screen reader user, and notice some major problems when using Parchment. Basically, the screen reader doesn't speak new text as it reaches the screen. Browsing the Google Code project, I notice there is acommit which experimented with ARIA and Live Regions. I don't know if this was reverted for some reason, but now it doesn't work.
I also noticed that Andrew Plotkin self-hosts his games in Parchment, and in there the screen reader behaves as it should (see any online game at <http://eblong.com/zarf/if.html>).

**Comment by curiousdannii** [comment]
I don't think Andrew has done anything to his version of Parchment, so I don't know why it would be any better. @erkyrath did you add any ARIA stuff to it?

**Comment by erkyrath** [comment]
No.

My site uses an old release of Parchment. I made a few changes to display better as a scrolling document with a header and footer, and I think to simplify the status line. But I didn't specifically invoke ARIA.

(It's this branch: https://github.com/erkyrath/parchment/tree/zarfsite )

**Comment by sukiletxe** [comment]
Then, it should be because of  him using an older version (I don't know JS so I can't browse the source). It reads almost everything. I can't remember when it didn't work.

**Comment by dfabulich** [comment]
Since this bug was filed, Zarf actually did an easy fix to Quixe that substantially fixed this bug there. https://github.com/erkyrath/glkote/commit/f938f035

The fix is to add three attributes to the "main" div:

````
<div
    aria-live="polite"
    aria-atomic="false"
    aria-relevant="additions"
>
```

**Comment by curiousdannii** [comment]
Thanks @dfabulich!
